content/docs/capabilities/directory-sync.mdx

```diff
@@ -0,0 +1,54 @@
+---
+id: directory-sync
+title: Directory Sync
+description: Directory Sync in Pomerium Enterprise allows you to import organizational directory data and external data sources you can use in authorization policies.
+keywords: [directory sync, idp, identity provider, groups, directory]
+---
+
+# Directory Sync
+
+**Directory Sync** is the process of synchronizing external directory data from your identity provider into the Enterprise Console. This document discusses how directory sync works in Pomerium and its use cases.
+
+:::enterprise
+
+**Directory Sync** is a Pomerium Enterprise feature. [Contact us](https://www.pomerium.com/enterprise-sales/) to upgrade today.
+
+:::
+
+## Directory sync in the Enterprise Console
+
+To start a directory sync in the Enterprise Console:
+
+1. Go to the **Identity Providers** tab
+1. Select your **Identity Provider**
+1. Next to [**IDP Options**](#idp-options), fill out the required fields
+1. In the [**Polling Min Delay**](#monitor-directory-sync) and [**Polling Max Delay**](#monitor-directory-sync) fields, enter the appropriate durations
+1. Select **SAVE SETTINGS** ![Selecting the Identity Providers tab in Enterprise Console for a directory sync](./img/directory-sync/directory-sync-idp-tab.gif)
+
+Once you save your settings, it may take awhile for the sync to complete. Go to [**Monitor directory sync**](#monitor-directory-sync) for more information.
+
+### Monitor directory sync
+
+The Enterprise Console polls the identity provider data source based on the durations defined in the **Polling Min Delay** and **Polling Max Delay** fields.
+
+See [**Identity Provider Min/Max Delay**](/docs/reference/identity-provider-settings#identity-provider-polling-minmax-delay) for more information on how to monitor directory sync.
+
+### IDP Options
+
+The requirements and instructions for directory sync vary depending on the identity provider. You can view the **IDP Options** for an identity provider in the Enterprise Console, or refer to the relevant identity provider guide for vendor-specific steps:
+
+- [Auth0](/docs/identity-providers/auth0)
```

Comment:
I think it would be even better if we could link directly to the directory sync instructions on each of these pages. I realize this is complicated by the current page structure, where the instructions are often inside a tab group.

What do you think about using the [`queryString`](https://docusaurus.io/docs/next/markdown-features/tabs#query-string) prop on the tab groups, so we can link to a specific tab? e.g.
 - add `queryString="groups"` to the relevant `Tabs` element on the Auth0 page
 - change the tab `value`s to something more URL-friendly (say `oss` and `enterprise`)
 - then change this link to `/docs/identity-providers/auth0?groups=enterprise#groups`

Reply:
Definitely an enhancement. Should we do that in a separate PR, as it requires updating several IdP guides?

Reply:
Sure, a separate PR would be fine.